test/utils.jl

```diff
@@ -3,13 +3,19 @@ using ForwardDiff
 using Zygote: hessian_dual, hessian_reverse
 
 @testset "hessian: $hess" for hess in [hessian_dual, hessian_reverse]
+  function f(x, bias)
+    hessian = hess(x->sum(x.^3), x)
+    return hessian * x .+ bias
```

Comment:
I think test functions should all have unique names, e.g. from issue numbers, and should not use exported symbols as variable names. Thus:
```suggestion
  function f1264(x, b)
    h = hess(x -> sum(x.^3), x)
    return h * x .+ b
```